Modules/IO/VTK/src/itkVTKImageIO.cxx

```diff
@@ -247,6 +249,7 @@ VTKImageIO::InternalReadImageInformation(std::ifstream & file)
     else if (text.find("origin") < text.length())
     {
       double origin[3];
+      std::locale currentLocale = std::locale::global(std::locale::classic());//save and reset old locale
       sscanf(text.c_str(), "%*s %lf %lf %lf", origin, origin + 1, origin + 2);
       for (unsigned int i = 0; i < m_NumberOfDimensions; i++)
```

Comment:
why not reset here ?

Reply:
Seems like I forgot to do it there. I'll add id into another commit.